### PR TITLE
panorama-security-policy-match test on entire panorama devices

### PR DESCRIPTION
**Pull request by esharf**
## Status
- [x] Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/43299

## Description
added support to test overall FW's when using panorama instance without supplying the `target` argument.
added support for the `vsys` argument (the current behavior is using vsys1 as default).

## Does it break backward compatibility?
- [x] No

## Must have
- [x] Tests

### CODE REVIEW

File: Packs/PAN-OS/Integrations/Panorama/Panorama.py [hunk 1]
```diff
@@ -4728,26 +4777,30 @@ def panorama_security_policy_match_command(args: dict):
...
-            'Contents': matching_rules,
-            'ReadableContentsFormat': formats['markdown'],
-            'HumanReadable': tableToMarkdown('Matching Security Policies:', ec_['Rules'],
-                                             ['Name', 'Action', 'From', 'To', 'Source', 'Destination', 'Application'],
-                                             removeNull=True),
-            'EntryContext': {"Panorama.SecurityPolicyMatch(val.Query == obj.Query)": ec_}
-        })
+        readable_output = readable_security_policy_match_outputs(context_list)
+
+        return_results(CommandResults(
```
Comment (yaakovi) [inline Packs/PAN-OS/Integrations/Panorama/Panorama.py:4801]: include the raw_response (a list of all the `matching_rules`)

File: Packs/PAN-OS/Integrations/Panorama/Panorama.py [hunk 2]
```diff
@@ -4711,13 +4712,61 @@ def prettify_query_fields(application: Optional[str] = None, category: Optional[
...
+                    final_vsys_s = map(lambda x: x['@name'], vsys_s_entry)
+                else:
+                    final_vsys_s = iter([None])
+            else:
+                final_vsys_s = vsys_s
+            for vsys in final_vsys_s:
+                yield device['serial'], vsys
+
+
+def format_eadable_security_policy_match_headers(hedear_name):
```
Comment (yaakovi) [inline Packs/PAN-OS/Integrations/Panorama/Panorama.py:4742]:
```suggestion
def format_readable_security_policy_match_headers(hedear_name):
```

File: Packs/PAN-OS/Integrations/Panorama/Panorama.py [hunk 3]
```diff
@@ -4711,13 +4712,61 @@ def prettify_query_fields(application: Optional[str] = None, category: Optional[
     return pretty_query_fields
 
 
-def panorama_security_policy_match_command(args: dict):
-    target = args.get('target')
-    if not VSYS and not target:
-        err_msg = "The 'panorama-security-policy-match' command is relevant for a Firewall instance " \
-                  "or for a Panorama instance, to be used with the target argument."
-        raise DemistoException(err_msg)
+def devices(targets=None, vsys_s=None):
```
Comment (yaakovi) [inline Packs/PAN-OS/Integrations/Panorama/Panorama.py:4715]: add docstring

**Review by yaakovi — APPROVED** [review]

**Comment by julieschwartz18** [comment]
@esharf - doc review complete for the Release Notes.

**Pull request merged**